tutorials/io/plot_20_reading_eeg_data.py

```diff
@@ -183,6 +183,27 @@
 sometimes from version to version. Please submit an issue, or
 pull request if you encounter a problem.
 
+Nihon Kohden EEG data (.EEG, .21E, .PNT, .LOG)
+======================================================================
+
+EEG data from the Nihon Kohden (NK) system can be read in using the
+:func:`mne.io.read_raw_nihon` function.
+
+The following file extensions are read in.
+- The ``.EEG`` file contains the actual raw EEG data.
+- The ``.PNT`` file contains the metadata related to the recording, such
+as the measurement date.
```

Comment:
```suggestion
  as the measurement date.
```
RST is picky, see how it rendered:

https://22536-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/io/plot_20_reading_eeg_data.html#nihon-kohden-eeg-data-eeg-21e-pnt-log